src/js/stores/NewsFeedStore.jsx

```diff
@@ -242,6 +420,25 @@ class NewsFeedStore extends EventEmitter {
         this.emit('error');
         break;
       }
+      case 'POST_FLAG_ERROR': {
+        this.emit('revert');
```

Comment:
Technically, yes?
Currently I optimistically update the UI so I use revert to tell it that something failed and it needs to regrab from the store because the expected change didn't happen. And then the error tells it to display an error